raiden-dapp/src/components/ConnectionManager.vue

```diff
@@ -152,8 +139,22 @@ export default class ConnectionManager extends Vue {
 </script>
 
 <style lang="scss" scoped>
+@import '@/scss/mixins';
+
 .connection-manager {
   font-size: 16px;
   line-height: 20px;
+
+  &__error-message {
+    min-height: 70px;
+
+    &--hidden {
+      visibility: hidden; // Make sure it still takes its height to avoid jumping
```

Comment:
Two solutions for the same problem I think. For me it was the straight forward to say it should be visible or not because this options was exactly made for my use-case. But if you prefer your solution I don't mind to change it. 🤷🏾

Reply:
I mean in this solution is not as intuitive and you even had to add a comment. But as I approved this PR this is just a minor comment and not a deal breaker. I'll let you decide which actions to take.